src/js/views/panel_view.js

```diff
@@ -78,20 +84,37 @@
                 h = this.model.get('height');
 
             this.update_resize(x, y, w, h);
+            this.render_labels();
             this.$el.removeClass('dragging');
         },
 
         update_resize: function(x, y, w, h) {
 
+            // If we have a panel border, need to adjust x,y,w,h on the page
+            // but NOT the w & h we use for img_css below.
+            var border = this.model.get('border');
+            var page_w = w;
+            var page_h = h;
+            if (border != undefined && !border.showBorder) {
```

Comment:
NB: need to check here if `border.showBorder` is positive (not negative).
If you want to try this in a more concise way, you might try https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
e.g. `if (border?.showBorder) {`
will only evaluate to `true` if border is defined AND `showBorder` is true.
No need to use that here, but I'm using it more now so just be aware in case you see it!

Reply:
I wasn't aware of this optional chaining. Thanks !